src/app/pages/credentials/certificates-dash/forms/certificate-acme-add.component.ts

```diff
@@ -139,7 +139,7 @@ export class CertificateAcmeAddComponent implements FormConfiguration {
   }
 
   preInit(entityForm: EntityFormComponent): void {
-    this.ws.call('acme.dns.authenticator.query').subscribe((authenticators: any[]) => {
+    this.ws.call('acme.dns.authenticator.query').subscribe((authenticators) => {
```

Comment:
Why was the type removed?

Reply:
I get that it's redundant but specific `any[]` is better than `any` I think

Reply:
When we specify types in `api-directory.interface.ts`, it will correctly pick up types in places where this API call is used.
In this case, I've typed `acme.dns.authenticator.query`, so `any[]` is no longer necessary.

Reply:
Yep. My bad. Forgot about that.